### PR TITLE
add instructions for Seurat from AtoMx

### LINKED ISSUE #90: Generate Seurat object from flat files
The [Seurat visualization post](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/posts/seurat-cosmx-basics/) starts from a Seurat object. Request is to add instructions for starting from flat files as well.

### PR DESCRIPTION
**Pull request by crwilliams11**
This PR addresses Issue #90 by adding instructions for how to export Seurat objects directly from AtoMx as well as how to load flat files into a Seurat object into the post on Seurat visualization of CosMx data.

### CODE REVIEW

File: posts/seurat-cosmx-basics/index.qmd [hunk 1]
```diff
@@ -59,11 +58,90 @@ Adjust globals option to avoid an error exceeding max allowed size. We've found
...
+#| label: "fig-flat-file-export-option"
+#| fig-cap: "Screenshot showing selection of flat files for export from AtoMx SIP."
+
+knitr::include_graphics("./figures/AtoMx_flatFileExport_image.png")
+```
+
+Once export is complete, download the files
+using your SFTP application (_e.g._, cyberduck, FileZilla, WinSCP).
+
+Load the flat files into a Seurat object using the `LoadNanostring` function from the Seurat package. Within the Seurat object, each slide is stored as a separate 'image' or 'fov'. This is an unfortunate naming convention difference between CosMx nomenclature and the Seurat package. What Seurat refers to as an ‘fov’ is what NanoString refers to as a slide. When you load flat files into a Seurat object, you need to name the Seurat fov / CosMx slide with the argument `fov = `. This name must be a character, not a numeric. Here we use "slide1", but more descriptive names are also possible.
```
Comment (eveilyeverafter) [inline posts/seurat-cosmx-basics/index.qmd]: ...between CosMx SMI nomenclature...

Comment (eveilyeverafter) [reply posts/seurat-cosmx-basics/index.qmd]: How does one refer to just a single SMI FOV then?

File: posts/seurat-cosmx-basics/index.qmd [hunk 2]
```diff
@@ -59,11 +58,90 @@ Adjust globals option to avoid an error exceeding max allowed size. We've found
...
+For convenience, you may want to save this Seurat object for future use.
+```{r}
+#| eval: false
+#| echo: true
+#| label: "save_seurat"
+
+saveRDS(seu.obj, file = "seurat_object.Rds")
+```
+
+If you have data from multiple CosMx slides, you can join them all into a single Seurat object with multiple images.
```
Comment (eveilyeverafter) [inline posts/seurat-cosmx-basics/index.qmd]: Add SMI after "CosMx" throughout.

**Review by eveilyeverafter — APPROVED** [review]
Minor comments then LGTM

File: posts/seurat-cosmx-basics/index.qmd (hunk 1 above)
Comment (crwilliams11) [reply posts/seurat-cosmx-basics/index.qmd]: "SMI" added and a note about how to plot single SMI FOVs added.

File: posts/seurat-cosmx-basics/index.qmd (hunk 2 above)
Comment (crwilliams11) [reply posts/seurat-cosmx-basics/index.qmd]: Added throughout in 613a712

**Pull request merged**